ubi_manifest/app/api.py

```diff
@@ -116,6 +117,38 @@ async def manifest_get(repo_id: str) -> DepsolverResult:
     raise HTTPException(status_code=404, detail=f"Content for {repo_id} not found")
 
 
+@router.delete(
+    "/manifest/{repo_id}",
+    status_code=200,
+    responses={
+        200: {
+            "description": "Succefully deleted manifest for repo_id entry",
+            "content": {
+                "application/json": {
+                    "example": {"detail": "manifest:foo-bar-repo entry deleted"}
+                }
+            },
+        },
+        404: {
+            "description": "Entry for repo_id not found",
+            "content": {
+                "application/json": {
+                    "example": {"detail": "Content for foo-bar-repo not found"}
+                }
+            },
+        },
+    },
+)
+async def manifest_delete(repo_id: str):
```

Comment:
Is the redis client designed for asyncio actually?  Seems not since you don't "await" it.  Then you probably shouldn't mark these endpoints as async - there is only a single thread running all the async requests, so if redis is slow it will slow down the entire web app.

Reply:
Thanks for pointing that out, I completely missed that asyncio thing.
The redis client we currently use is not async but it looks that async api should be soon provided. The redis-py is going to merge https://aioredis.readthedocs.io/ , I think I'll stick no-async approach for the time being and remove the async keywords there.